resources/views/Admin/user/seeker/userInfo.blade.php

```diff
@@ -1,120 +1,116 @@
+                                            <img src="{{asset('admin/images/avatar-default.svg')}}" alt="" id="user-pic" class="card-img rounded-circle bg-light" />
+                                            <i class="mdi mdi-link-variant-plus fs-4 position-absolute bottom-0 start-75 translate-middle btn btn-sm d-none" id="user_pic_change_icon"></i>
                                         </div>
-                                        <div class="col-6 my-auto">
-                                            <h3>
-                                                {{$user->firstName}} {{$user->secondName}}
-                                            </h3>
-                                            <p class="card-subtitle card-subtitle-dash">
-                                                joined {{ceil(\Carbon\Carbon::now()->diffInMonths(\Carbon\Carbon::parse($user->created_at)))}}
-                                                months ago.
-                                            </p>
-                                            <div class="row justify-content-start g-2">
-                                                <a href="#" class="col-auto">
-                                                    <i class="mdi mdi-linkedin text-dark"></i>
-                                                </a>
-                                                <a href="#" class="col-auto">
-                                                    <i class="mdi mdi-instagram text-dark"></i>
-                                                </a>
-                                                <a href="#" class="col-auto">
-                                                    <i class="mdi mdi-facebook text-dark"></i>
-                                                </a>
-                                                <a href="#" class="col-auto">
-                                                    <i class="mdi mdi-twitter text-dark"></i>
-                                                </a>
-                                            </div>
+                                    </div>
+                                    <div class="col-6 my-auto">
+                                        <h3>
+                                            {{$user->firstName}} {{$user->secondName}}
+                                        </h3>
+                                        <p class="card-subtitle card-subtitle-dash">
+                                            joined {{ceil(\Carbon\Carbon::now()->diffInMonths(\Carbon\Carbon::parse($user->created_at)))}}
+                                            months ago.
+                                        </p>
+                                        <div class="row justify-content-start g-2">
+                                            @foreach ($user->SocialMedia as $socialMedia)
+                                            @if ($socialMedia->mediaName === 'LinkedIn')
```

Comment:
This comparison is a little risky.
- Admins can make a mistake and change the social media names from LinkedIn to linkedIn, this code will not able to solve this little bugs 

My advice to you is to make a little optimization, add social media name to enums
This will force db admin to enter specific values only 
make your comparison valid

Reply:
Dear Nourhan,
Thank you so much for your kind and sweet words 😘😘. You are an amazing mentor and we have all learned a lot and benefited from your insightful and valuable comments. I have made the required changes and pushed the request. Thanks again😍. Samar